Browser_IDE/splashKitOnlineEnvParams.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+function findGetParameter(parameterName, _default=null) {
+    location.search.substr(1).split("&").forEach(function (piece) {
+      let tmp = piece.split("=");
+      if (tmp[0] === parameterName)
+          return decodeURIComponent(tmp[1]);
+    });
+
+    return _default;
+}
```

Comment:
I can't find any place in which this function is actually used. Am I missing one?

Reply:
It _was_ used... You can probably tell that `splashKitOnlineEnvParams.js` was a bit rushed 😅